share/mk/bsd.compat.mk

```diff
@@ -144,13 +144,13 @@ COMPAT_RISCV_MARCH=	rv64ima
 COMPAT_RISCV_MARCH:=	${COMPAT_RISCV_MARCH}fd
 .endif
 COMPAT_RISCV_MARCH:=	${COMPAT_RISCV_MARCH}c
-.if ${COMPAT_ARCH:Mriscv*c*} || (defined(HAS_COMPAT) && ${HAS_COMPAT:MCHERI})
+.if ${COMPAT_ARCH:Mriscv*c*} || (defined(HAS_COMPAT) && ${HAS_COMPAT:M64C})
 COMPAT_RISCV_MARCH:=	${COMPAT_RISCV_MARCH}xcheri
 .endif
 .endif
 
 # Common CHERI flags
-.if defined(HAS_COMPAT) && ${HAS_COMPAT:MCHERI}
+.if defined(HAS_COMPAT) && ${HAS_COMPAT:M64C}
 LIB64CCFLAGS+=	-DCOMPAT_CHERI
```

Comment:
Hmm, what about COMPAT_CHERI?

Reply:
It's COMPAT_32/64BIT not COMPAT_(LIB)32/64; unclear what the equivalent is so left it alone

Reply:
Ok, I'm fine with leaving it as.